ORBIT/manager.py

```diff
@@ -323,11 +323,21 @@ def compile_input_dict(cls, phases):
             "project_lifetime": "yrs (optional, default: 25)",
             "discount_rate": "yearly (optional, default: .025)",
             "opex_rate": "$/kW/year (optional, default: 150)",
-            "construction_insurance": "$/kW (optional, default: 44)",
-            "construction_financing": "$/kW (optional, default: 183)",
-            "contingency": "$/kW (optional, default: 316)",
-            "commissioning": "$/kW (optional, default: 44)",
```

Comment:
I like the idea of flagging when a parameter is deprecated with a warning print, but I dont think it is positive for the users to keep parameters that are not used anymore at config["project_parameters"], I believe it could be misleading. 

I will work on the warning and discuss what is best on Teams tomorrow